src/routes/Index.svelte

```diff
@@ -1 +1,14 @@
+<script>
+  import SomeComponent from "../components/SomeComponent.svelte";
+  import SomeGraph from "../components/SomeGraph.svelte";
+</script>
+
 <h2>Home</h2>
+
+<p>This is a component:</p>
+
+<SomeComponent />
```

Comment:
Funny that this component has an `on:click` but doesn't appear to be wired to do anything?

Reply:
Seems like a good issue to tackle in a followup